packages/web-app-files/src/components/Search/List.vue

```diff
@@ -276,11 +282,8 @@ export default defineComponent({
       () => unref(route).query,
       (newVal, oldVal) => {
         const filters = ['q_fullText', 'q_tags']
-        const isChange =
-          newVal?.term !== oldVal?.term ||
-          filters.some((f) => newVal[f] ?? undefined !== oldVal[f] ?? undefined)
 
-        if (isChange && isLocationCommonActive(router, 'files-common-search')) {
```

Comment:
Why did you remove all that code? It now fires a search on every route change, we should avoid that. Wouldn't it be sufficient to add a check if the `useScope` param has changed?